language-guide/concurrency.md

```diff
@@ -275,12 +262,9 @@ let photos = await withTaskGroup(of: Optional<Data>.self) { group in
 上記のコードでは、以前のバージョンからいくつかの変更が加えられています:
 
 - 各タスクは [`TaskGroup.addTaskUnlessCancelled(priority:operation:)`](https://developer.apple.com/documentation/swift/taskgroup/addtaskunlesscancelled(priority:operation:))メソッドを使用して追加され、キャンセル後に新しい作業が開始されるのを防ぎます
-- `addTaskUnlessCancelled(priority:operation:)` を呼び出すたびに、コードは新しい子タスクが追加されたことを確認します。グループがキャンセルされた場合、`added` の値は `false` になります。上記の例の場合、コードは追加の写真をダウンロードしようとするのを止めます
 - 各タスクは、写真のダウンロードを開始する前に、キャンセルされたかどうかをチェックします。キャンセルされた場合、タスクは `nil` を返します
 - 最後に、タスクグループは結果を集める際に `nil` 値をスキップします。`nil` を返すことでキャンセルを処理することは、タスクグループが完了した作業を破棄する代わりに、部分的な結果(キャンセル時にすでにダウンロードされていた写真)を返すことができるということです
 
- > NOTE: タスクの外側からタスクがキャンセルされたかどうかを確認するには、型プロパティの代わりに [`Task.isCancelled` インスタンスプロパティ](https://developer.apple.com/documentation/swift/task/iscancelled-swift.property)を使用します。
```

Comment:
こちらも原文の↓に対応するのかなと思います。

Note: To check whether a task has been canceled from outside that task, use the [Task.isCancelled](https://developer.apple.com/documentation/swift/task/iscancelled-swift.property) instance property instead of the type property.

https://github.com/apple/swift-book/blob/main/TSPL.docc/LanguageGuide/Concurrency.md#task-cancellation